src/control-plane/backend/lambda/api/router/reporting.ts

```diff
@@ -89,6 +89,12 @@ reporting_project.post(
     return reportingServ.cleanQuickSightResources(req, res, next);
   });
 
+reporting_project.get(
```

Comment:
Why do we need a separate API for checking if Studio is enabled for the project?

Reply:
On the four pages of the Analysis Studio, call this API uniformly to first determine whether the current project is enabled. If it is enabled, then request other API. Otherwise, do not call other interfaces to avoid epactal API calls. At the same time, it can also avoid unfriendly prompts such as API parameter errors.